java/ql/test/TestUtilities/InlineExpectationsTest.qll

```diff
@@ -202,6 +202,7 @@ private predicate getAnExpectation(
       column = TDefaultColumn() and
       exists(int end |
         end = getEndOfColumnPosition(0, content) and
+        not exists(string name | exists(TNamedColumn(name)) and content.indexOf(name + ":") < end) and
```

Comment:
Doesn't this check belong in `getEndOfColumnPosition`? It already does a very similar check that checks the start position.

Reply:
It seems to me that `getEndOfColumnPosition` is doing the right thing -- it's identifying that the next column is the next column keyword or the end of the string.

The problem is that `MISSING: abc` could previously match both this disjunct (`column` = `TDefaultColumn()`, `end` = EOS, `expectation` = `abc`) and the other disjunct (`column` = `TNamedColumn("MISSING")`, `start` = `len("MISSING: ")`, `end` = EOS, `expectation` = `abc`)

Reply:
I suppose alternatively we could replace `regexpFind` with a `regexpMatch` consuming only whitespace up front, so `MISSING: abc` would fail to match due to the `:` character

Reply:
Ah, right. I see what you mean. I think replacing `regexpFind` with `regexpMatch` is morally the right thing to do, but I don't dislike this solution enough to argue against your fix. 👍

Reply:
I tried using `regexpMatch`, but it's awkward because it's using `regexpFind` in order to bind more than one chunk that matches `expectationRegexp()` -- for example, in `// $ a=b c=d` that `regexpFind` is supposed to match twice, once starting at `a` and again starting at `c`.